File: php/examples/GetService.php
<?php

use Qcloud\Cos\Client;
use Qcloud\Cos\Exception\ServiceResponseException;

class GetService
{
    private $cosClient;

    private $uploadId;
    private $eTag;
    private $versionId;

    // 获取存储桶列表
    protected function getService() {
        $cosClient = $this->cosClient;
        //.cssg-snippet-body-start:[get-service]
        try {
            //请求成功
            $result = $cosClient->listBuckets();
            print_r($result);
        } catch (\Exception $e) {
            //请求失败
            echo($e);
        }
        
        //.cssg-snippet-body-end
    }

	//.cssg-methods-pragma

    protected function init() {
        $secretId = "COS_SECRETID"; //"云 API 密钥 SecretId";
        $secretKey = "COS_SECRETKEY"; //"云 API 密钥 SecretKey";
        $region = "COS_REGION"; //设置一个默认的存储桶地域
        $this->cosClient = new Qcloud\Cos\Client(
            array(
                'region' => $region,
                'schema' => 'https', //协议头部，默认为http
                'credentials'=> array(
                    'secretId'  => $secretId ,
                    'secretKey' => $secretKey)));
    }

    public function mGetService() {
        $this->init();

        // 获取存储桶列表
        $this->getService();

	    //.cssg-methods-pragma
    }
}
?>

File: Python/examples/get_service.py
# -*- coding=utf-8

from qcloud_cos import CosConfig
from qcloud_cos import CosS3Client
from qcloud_cos import CosServiceError
from qcloud_cos import CosClientError

secret_id = 'COS_SECRETID'     # 替换为用户的secret_id
secret_key = 'COS_SECRETKEY'     # 替换为用户的secret_key
region = 'COS_REGION'    # 替换为用户的region
token = None               # 使用临时密钥需要传入Token，默认为空,可不填
config = CosConfig(Region=region, SecretId=secret_id, SecretKey=secret_key, Token=token)  # 获取配置对象
client = CosS3Client(config)

# 获取存储桶列表
def get_service():
    #.cssg-snippet-body-start:[get-service]
    response = client.list_buckets(
    )
    
    #.cssg-snippet-body-end

#.cssg-methods-pragma


# 获取存储桶列表
get_service()

#.cssg-methods-pragma